wandb/sdk/launch/agent/agent.py

```diff
@@ -563,6 +563,8 @@ def _thread_run_job(
         job_tracker: JobAndRunStatusTracker,
     ) -> None:
         project = create_project_from_spec(launch_spec, api)
+        project.queue_name = job_tracker.queue
```

Comment:
Don't we want to store the full URI of the queue here? I'm pretty sure this is just the queue name